### ISSUE TITLE
Missing publishing steps for CSIRO DAP

### ISSUE THREAD

**Issue by paolap** [ACDguide/Governance#88]
At the moment publish-csior-dap is juts a copy of what we wrote in the publishing -option. We need some detail, this comment was in the actual page, I'm copying it here and remove it form there.

!!!COMMENT: at the moment this is identical to what we have in the Publishing option pages for CSIRO. I'm leaving this here for the moment in case we want to add something more, like:
 - summary of the procedure with pros and cons?
 - Something which is not covered in the official documentation but could be useful?
 - a reminder to follow discipline specific best practices even if they aren't necessarily required?

**Comment by hot007** [comment]
Do you want me to write that or @organisergirl ? ie researcher perspective or provider perspective? I'm happy to take it as an action but just thought I'd check! Or I could do a first pass and Katie edit it?

**Comment by paolap** [comment]
It should be a step by step how do I publish on DAP?
If you only do a provider point of view we might miss on things which aren't required, for example applying formats etc. 
However, if there's enough in the DAP documentation for staff and we can refer them to the generic public procedure to take care of what might not be covered there, you could just skip this page completely and clarify how a user can use the generic procedure + the detailed  DAP documentation directly in the CSIRO tab in publishing-options. I couldn't do that for CLEX as there's no documentation on how to publish at NCI and we have to create our own workflow.

**Comment by organisergirl** [comment]
I’d prefer that we don’t duplicate any documentation as it means that we have to keep multiple places up to date.
The DAP publication documentation is only available to CSIRO staff.

**Comment by paolap** [comment]
I don't think that it's only available to staff is an issue because this page should potentially cover the actual steps to publish which are only relevant to Csiro staff. So probably removing this page and updating the "Cairo publishing options" it's the way to go. I'm happy to remove this file now before the next push. The info at the moment is exactly what's in the options so there wouldn't be any loss.
